server/src/test/java/org/elasticsearch/cluster/coordination/NodeLeftExecutorTests.java

```diff
@@ -128,13 +128,11 @@ public void testPerNodeLogging() {
                     "node-left: [" + nodeToRemove.descriptionWithoutAttributes() + "] with reason [test reason]"
                 )
             );
-            assertNull(
-                PlainActionFuture.<Void, RuntimeException>get(
-                    future -> clusterService.getMasterService()
-                        .createTaskQueue("test", Priority.NORMAL, executor)
-                        .submitTask("test", new NodeLeftExecutor.Task(nodeToRemove, "test reason", () -> future.onResponse(null)), null)
-                )
-            );
+            final var latch = new CountDownLatch(1);
+            clusterService.getMasterService()
+                .createTaskQueue("test", Priority.NORMAL, executor)
+                .submitTask("test", new NodeLeftExecutor.Task(nodeToRemove, "test reason", latch::countDown), null);
+            safeAwait(latch);
```

Comment:
Again here, I'm interested to understand when to use `safeAwait(CheckedConsumer)` vs the above.

Reply:
Similarly: it's a bit less nesting, and indicates that we're not capturing any failures here. Don't really have a strong preference tho.